mscore/editelement.cpp

```diff
@@ -94,6 +94,9 @@ void ScoreView::startEditMode(Element* e)
             }
       if (score()->undoStack()->active())
             score()->endCmd();
+      // Restart edit mode to reinit edit values
+      if (editMode())
+            changeState(ViewState::NORMAL);
```

Comment:
I tried before endcmd and it causes a segfault. I will update the PR to unify these changes then, and make sure it's a generic `editMode` instead of just `textEditMode`.